src/components/shared/BasePopup/base.popup.module.scss

```diff
@@ -34,8 +34,8 @@
 }
 
 .popup {
-  max-width: 410px;
-  min-width: 320px;
+  max-width: 600px;
+  min-width: 450px;
```

Comment:
that needs to, don't you think that this numbers are better ones?

Reply:
BasePopup is used for errors popup too, and there is no need to create big pop up, maybe it would be better to create some wrapper for forms based on basePopUp and set this style there? Or to pass it into component, like "classes".